libr/core/cmd_write.c

```diff
@@ -1405,21 +1405,23 @@ static int cmd_write(void *data, const char *input) {
 		case 'o': // "wao"
 			if (input[2] == ' ') {
 				char *op = r_str_trim (strdup (input + 3));
-				r_core_hack (core, op);
-				free (op);
+				if (op) {
+					r_core_hack (core, op);
+					free (op);
+				}
 			} else {
 				r_core_hack_help (core);
 			}
 			break;
 		case ' ':
 		case '*': {
-			const char *file = input[1]=='*'? input+2: input+1;
```

Comment:
There should really be a mechanical way to retrieve the nth argument, which will auto benefit auto completion (which currently encodes the kind of the first argument, which is not generic).